### DISCUSSION TITLE
Specific routes for NativePHP app

### DISCUSSION THREAD

**Discussion by maurocasas** [NativePHP/laravel#155] [Ideas]
I'm creating this feature request so I can contribute with it.

I believe it'd be interesting to be able to have both the NativePHP app and web/API under the same project by dividing the routes in projects of lesser complexity.

**Comment by simonhamp** [comment]
You could do this right now, just split your routes into a `routes/native.php` file and makes sure any windows you create reference those routes instead

In your `App\Providers\RouteServiceProvider::boot()`, add something like:

```diff
            Route::middleware('web')
                ->group(base_path('routes/web.php'));
+
+            Route::middleware('web')
+                ->name('native.')
+                ->prefix('native')
+                ->group(base_path('routes/native.php'));
```

Create a `routes/native.php` file and place your routes there.

Then make sure any Windows you create in NativePHP reference those routes explicitly, for example:

```php
Window::open()
    ->url(url('native/')
    ...

// OR

Window::open()
    ->route('native.home')
    ...
```
